collabora_online.info.yml

```diff
@@ -1,6 +1,6 @@
 name: Collabora Online
 description: Integrate Collabora Online for document viewing and editing.
-package: Custom
+package: Collabora online
```

Comment:
Should we capitalize the "Online"? No opinion myself.

Reply:
Yep it's capitalised https://www.drupal.org/project/collabora_online

Reply:
Changed